src/main/app/common/calculate-monthly-income-expense/incomeExpenseSource.ts

```diff
@@ -57,6 +59,32 @@ export class IncomeExpenseSource {
       toIncomeExpenseScheduleOrUndefined(schedule)
     )
   }
+
+  static fromClaimIncome (income: Income): IncomeExpenseSource {
+    if (!income) {
+      return undefined
+    }
+
+    const schedule: string = income.frequency ? income.frequency : undefined
```

Comment:
do we need a conditional operator here? It would be same otherwise

Reply:
Good spot. Removed.